src/shared/navgen/nav.cpp

```diff
@@ -339,11 +345,12 @@ static void LoadPatchTris( std::vector<float> &verts, std::vector<int> &tris ) {
 	 */
 
 	// calculate bounds of all verts
+	vec3_t mins, maxs;
 	rcCalcBounds( &verts[ 0 ], verts.size() / 3, mins, maxs );
 
 	// convert from recast to quake3 coordinates
-	recast2quake( mins );
-	recast2quake( maxs );
+	std::swap( mins[ 1 ], mins[ 2 ] );
+	std::swap( maxs[ 1 ], maxs[ 2 ] );
```

Comment:
I like the `recast2quake` call better, but this is mostly cosmetic

Reply:
recast2quake currently lives in sgame and it didn't seem worth the restructuring effort

Reply:
fair enough